packages/expo-camera/android/src/main/java/expo/modules/camera/next/Options.kt

```diff
@@ -0,0 +1,22 @@
+package expo.modules.camera.next
+
+import android.media.CamcorderProfile
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+
+class PictureOptions : Record {
```

Comment:
```suggestion
data class PictureOptions : Record {
```

Reply:
Not opposed to this but we don't use any features provided by a data class. Curious why we would use them? Consistency is the only reason that comes to mind.

Reply:
we have some existing code that uses data class already, e.g. https://github.com/expo/expo/blob/f9d4a806e0f8b21bf8511456d9db7e2920f51cd0/packages/expo-sharing/android/src/main/java/expo/modules/sharing/SharingOptions.kt#L6

the reason i would like to propose those options as data classes is because of the default implementation of `equals`. and our options usage looks more sense of using data classes to be honest.